src/components/Select/Select.tsx

```diff
@@ -108,10 +110,6 @@ export const Select: React.FC<SelectProps> = ({
     onChange && onChange(state.value);
   }, [onChange, state.value]);
 
-  useUpdateEffect(() => {
-    value && dispatch({ type: 'SELECT', payload: { value } });
-  }, [value]);
```

Comment:
Why delete this? I think we still want to be able to control the `value` from props.

Reply:
this hasn't had to get deleted, I don't know what happened